src/main/java/clzzz/helper/commons/util/DateTimeUtil.java

```diff
@@ -10,7 +11,7 @@
     // datetime-related
     public static final String DATETIME_PATTERN = "d/M/yyyy HHmm";
     public static final DateTimeFormatter DATETIME_FORMAT =
-            DateTimeFormatter.ofPattern(DATETIME_PATTERN);
+            DateTimeFormatter.ofPattern(DATETIME_PATTERN).withResolverStyle(ResolverStyle.LENIENT);
```

Comment:
Why not set this to strict and then check day against length of month?
LENIENT doesn't make sense to me since it simply means #130 will be accepted rather than rounded inward.
STRICT checks outer range so it won't reject 29 to 31, if I'm not misinterpreting the documentation.

Reply:
https://howtodoinjava.com/java/date-time/resolverstyle-strict-date-parsing/
Actually just setting it to STRICT does all the checking for you 🤔

Reply:
because if I set to strict i will not allow input like "12/03/2020" since it does not match "d/m/yyyy". That's why I need to put linient first and the check if the date is valid by myself.

Reply:
dd/MM/yyyy?

Reply:
This would mean we can't use the same format for output if we don't want the padding, but I don't think either the number of formats or the presence of the padding would be an issue.

Reply:
Another reason why I dont use STRICT is that both invalid date and invalid format will throw the same exception when parsing the string to localdate. Anyways I need to differentiate between them again and throw different error messages. Don't think it will be more convenient to use STRICT...

Reply:
Can’t argue with that